tests/functional-tests/readme.md

```diff
@@ -6,22 +6,25 @@ This is an example of using Browserstack in conjunction with Geb/Spock tests as
 
 It works out of the box with Chrome, Edge, and Firefox on Windows 10, and can be readily configured to support many more
  browsers (even mobile devices) by adjusting the configuration in `src/test/resources/GebConfig.groovy`.
- 
+
  It works equally well on Windows, Mac, and Linux hosts since tests run remotely, and it works well within an Openshift
   or Jenkins pipeline.
- 
+
+## Prerequisites
+1. Start the application under test
+2. Make sure you have Java installed (Min. Version 1.8)
+3. Make sure you have Gradle 4.2.1 installed: `choco install gradle --version=4.2.1`
```

Comment:
Would be good to note that the choco command is windows specific for non-windows audiences.